pkg/internal/cmdparams/doc.go

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright 2022 The Sigstore Authors.
```

Comment:
if new file: 2022 -> 2023

Reply:
changed in [42b38a4](https://github.com/sigstore/timestamp-authority/pull/474/commits/42b38a49396690afa26c1c224e7d6542b56721ea)